dips/dip-10.md

```diff
@@ -72,7 +72,7 @@ The format of the command is:
    "_ObjectType": "CommandRequestObject",
     "command_type": "SenderInitiatedReferenceIDExchange",
     "command": {
-	    "_ObjectType": "ReferenceIDCommand",
+	    "_ObjectType": "SenderInitiatedReferenceIDExchange",
```

Comment:
I thought you proposed to use `SenderInitiatedDiemIDExchange`

Reply:
DiemID is one of many identifiers that could be used, so it isn't quite accurate.

Reply:
I heard that we want to move away from using Diem ID everywhere :D I don't mind either `SenderInitiatedReferenceIDExchange` or `SenderInitiatedDiemIDExchange`, it's more general vs more specific.. @davidiw can help make final call please